dataplane/pkg/common/egress_interface.go

```diff
@@ -92,17 +96,20 @@ func parseProcFile(reader *bufio.Reader) (string, net.IP, error) {
 	return "", nil, errors.New("Failed to locate default route...")
 }
 
-func parseGatewayIP(defaultGateway string) net.IP {
+func parseGatewayIP(defaultGateway string) (net.IP, error) {
```

Comment:
@haiodo I'm not sure, but seems like `parseGatewayIP` could be replaced with `net.ParseIP`. Please correct me if I'm wrong

Reply:
@lobkovilya not sure ParseIP will work here from the doc string:

```The string s can be in dotted decimal ("192.0.2.1")```

Reply:
@pringlewood Do you mean this case https://play.golang.org/p/ZuCQ-EXbVsI?

Reply:
@denis-tingajkin yup, I updated the example to try and parse a string as per the file: 

https://play.golang.org/p/j5IQ1tj_vlm

Reply:
@pringlewood got it